src/analysis/normed_space/basic.lean

```diff
@@ -463,11 +463,21 @@ lemma mul_left_bound {α : Type*} [normed_ring α] (x : α) :
   ∀ (y:α), ∥add_monoid_hom.mul_left x y∥ ≤ ∥x∥ * ∥y∥ :=
 norm_mul_le x
 
+/-- In a normed ring, the left-multiplication `add_monoid_hom` is continuous. -/
+lemma mul_left_continuous {α : Type*} [normed_ring α] (x : α) :
+  continuous (add_monoid_hom.mul_left x) :=
+(add_monoid_hom.mul_left x).continuous_of_bound ∥x∥ (mul_left_bound x)
```

Comment:
I guess a `normed_ring` is a `topological_monoid`, so you can get this as `continuous_const.mul continuous_id`.

Reply:
Thanks, I changed the proof and moved the lemma to `topology.algebra.ring`.